src/dlgTriggerEditor.cpp

```diff
@@ -603,6 +604,14 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     }
     showHiddenVars = false;
     widget_searchTerm->updateGeometry();
+
+    if (mAutosaveInterval > 0) {
+#if QT_VERSION < QT_VERSION_CHECK(5, 9, 0)
+        startTimer(mAutosaveInterval * 1000);
```

Comment:
Isn't that a 1 second interval instead of 1 minute?

Reply:
This is why I want Qt 5.9... no silly bugs like this. Fixed.